src/main/webapp/app/pages/userManagement/UserDetailsPage.tsx

```diff
@@ -85,7 +123,7 @@ export default class UserDetailsPage extends React.Component<{
   }
 
   private getStatus(activated: boolean) {
```

Comment:
This status func is dup from user management. I didn't compare these two files, there might be more. Let's don't worry about that yet until we figure out a plan for these two pages